<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_enviar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enviar_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nombre">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>27</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del Cliente</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_nombre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>517</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumBlockCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_dni">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DNI del Cliente</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_dni">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>113</y>
     <width>517</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumBlockCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QMessageBox" name="messageBox">
   <property name="windowTitle">
    <string>Alerta</string>
   </property>
   <property name="text">
    <string>Por favor, complete todos los campos.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
